packages/new_stats/README.md

```diff
@@ -0,0 +1,45 @@
+# Grid Stats
+
+## Introduction
+
+Grid Stats is a web application created to replace explorer.threefold.io with a simple page showing a capacity map + statistics. In the map, you get to see node distribution with each node location highlighted. While in Statistics section, general details about the nodes are being displayed like online nodes, farms count, total storage used, gateways ...etc.
+Also, you can select one or more networks to get their stats.
+
+## Installation
+
+- **Prerequisite**
+
+  - Nodejs 16^
```

Comment:
Some SDK packages require Nodejs 18, so I think it would be better if we used 18^ as it's in the same repo.